web/src/testgrid-failures-summary.ts

```diff
@@ -0,0 +1,74 @@
+import { LitElement, html, css } from 'lit';
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+import { customElement, property, state } from 'lit/decorators.js';
+import { TabSummaryInfo } from './testgrid-dashboard-summary';
+import { map } from 'lit/directives/map.js';
+
+@customElement('testgrid-failures-summary')
+export class TestgridTabTable extends LitElement {
```

Comment:
It's better to keep the class name the same the custom element name. i.e
```
export class TestgridFailuresSummary extends LitElement
```

Reply:
Done!